src/librustc-builtins/builtins

```diff
@@ -0,0 +1 @@
+../compiler-rt/lib/builtins
```

Comment:
You mean having the `build.rs` directly use the `$RUST_REPO/src/compiler-rt/lib/builtins` directory?